packages/manager/modules/navbar/src/notifications-menu/controller.js

```diff
@@ -87,10 +124,18 @@ export default class NotificationsCtrl {
       .catch(() => undefined);
   }
 
-  openMenu() {
-    this.atInternet.trackClick({
-      name: 'navbar::action::notifications',
-      type: 'action',
-    });
+  onTriggerClick(open) {
+    if (open) {
+      // Handle the click outside the menu
+      this.$document.on('click', this.readAllNotifications);
+
+      this.atInternet.trackClick({
+        name: 'navbar::action::notifications',
+        type: 'action',
+      });
+    } else {
+      // We unbind the click event in this function
+      this.readAllNotifications();
+    }
```

Comment:
This looks complicated for just handling a click on opening a menu

Reply:
What's complicated ? It supports the click on the trigger button and when we close the menu by clicking outside the panel.

The most complicated part I can see, is the click binding on the `$document`.
We need to unbind the function to avoid polluting `$document` with unnecessary function.

Reply:
Is this a wanted behaviour though ? Because the user panel behaves differently and close only if you click on the button in the navbar. Shouldn't we have a consistency between those two behaviours ?

Reply:
Yes, like the dropdown component, it closes the menu/panel when we click outside. It's implemented in the navbar: 
https://ovh.github.io/ovh-ui-kit/?path=/story/legacy-components-navbar-webcomponent--simple

I agree that we should have a consistency between all panels of the navbar. I could take a look at the user panel if you want.